editor/editor_properties_array_dict.cpp

```diff
@@ -517,6 +539,52 @@ void EditorPropertyArray::drop_data_fw(const Point2 &p_point, const Variant &p_d
 		emit_changed(get_edited_property(), array, "", false);
 		update_property();
 	}
+
+	if (drop_type == "nodes") {
+		Array nodes = drag_data["nodes"];
+		Node *base_node = get_base_node();
+
+		for (int i = 0; i < nodes.size(); i++) {
+			Node *node = get_node(nodes[i]);
+			NodePath path = node->get_path();
```

Comment:
```suggestion
			NodePath path = nodes[i];
			Node *node = get_node(path);
```

maybe you should also rename all your `nodes` variables to `node_pathes` or something like that to make clear they contain pathes and not the nodes directly..